lib/events/complete.go

```diff
@@ -438,6 +457,28 @@ loop:
 	if err := u.cfg.AuditLog.EmitAuditEvent(ctx, sessionEndEvent); err != nil {
 		return trace.Wrap(err)
 	}
+
+	if !isPTYSession {
+		return nil
+	}
```

Comment:
Why do we differentiate between PTY sessions here but not elsewhere? Are we missing database summaries as a result?

Reply:
Other places do not differentiate because they also support database sessions.
session completer does not support database sessions so we can safely ignore everything that's not PTY - SessionStart/SessionEnd events